### ISSUE TITLE
Android - QAB - On opening QAB, the upper part flickers once and opens

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#48116]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.25
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Launch app
2. Tap fab -- submit expense
3. Enter an amount and tap next
4. Select a user
5. Submit expense
6. Navigate to LHN
7. Tap fab
8. Tap QAB

## Expected Result:
On opening QAB, the upper part must not flicker and open

## Actual Result:
On opening QAB, the upper part flickers once and opens

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/user-attachments/assets/0b148ab3-45a4-43bc-b6de-3a86ad96d7bd

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
@mallenexpensify  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by mallenexpensify** [comment]
Thanks @tsa321 , I'm going to put this on hold then, pending #46352

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by mallenexpensify** [comment]
Held

**Comment by mallenexpensify** [comment]
Posted on the issue we're held on
- https://github.com/Expensify/App/issues/46352#issuecomment-2369825515

**Comment by mallenexpensify** [comment]
Thanks @tsa321 for the headsup.  Let's wait til your PR hits production then we'll test again and determine the next best step (since the bug is just a lil flicker and does't affect functionality)

**Comment by mallenexpensify** [comment]
- https://github.com/Expensify/App/issues/46352

Above is off hold, throwing `restest-weekly` on here.

**Comment by mallenexpensify** [comment]
Waiting on retest, I'm off the next week, will revisit once back.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Second week)

**Comment by mallenexpensify** [comment]
Not reproducible, going to close
